### PR TITLE
Updated TestContext information

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

File: docs/articles/nunit/writing-tests/attributes/testcase.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ TestCaseAttribute supports a number of additional named parameters:
 * **Reason** specifies the reason for not running this test case. Use in conjunction with **Explicit**.
 * **TestName** provides a name for the test. If not specified, a name is generated based on the method name and the
   arguments provided. See [Template Based Test Naming](xref:templatebasedtestnaming).
-* **TestOf** specifies the Type that this test is testing
+* **TestOf** specifies the Type that this test is testing (this is not used within NUnit)
```
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/attributes/testcase.md:41]:
```suggestion
* **TestOf** specifies the Type that this test is testing (this is not used within NUnit during test execution, but may serve a purpose for the test author)
```

Minor wording suggestion (assuming I got your intent correct)

File: docs/articles/nunit/writing-tests/TestContext.md [hunk 2]
```diff
@@ -143,9 +148,25 @@ Gets a representation of the current test, with the following properties:
 
 * **ID** - The unique Id of the test
 * **Name** - The name of the test, whether set by the user or generated automatically
+* **Namespace** - The namespace of the test
+* **DisplayName** - The display name of the test
 * **FullName** - The fully qualified name of the test
 * **MethodName** - The name of the method representing the test, if any
-* **Properties** - An `IPropertyBag` of the test properties
+* **Method** - The IMethodInfo of the test method
+* **Type** - The Typeinfo of the test
```
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/TestContext.md:156]:
```suggestion
* **Type** - The `TypeInfo` of the test
```

I think this is correct? (if so, it'll let you remove the cSpell dictionary addition as well I think

File: docs/articles/nunit/writing-tests/TestContext.md [hunk 3]
```diff
@@ -143,9 +148,25 @@ Gets a representation of the current test, with the following properties:
...
+* **ClassName** - The classname of the test
+* **Arguments** - The arguments as a list of objects, if any.
+* **ExpectedResult** - The expected result, if that is set for the test
+* **Properties** - An `IPropertyBag` of the test properties. Note: The Get method ONLY returns the first property
```
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/TestContext.md:160]:
```suggestion
* **Properties** - An `IPropertyBag` of the test properties. Note: The `Get` method ONLY returns the first property
```

**Review by SeanKilleen — APPROVED** [review]
Generally LGTM -- a few questions but once addressed I'm fine if this gets merged 👍

**Comment by OsirisTerje** [comment]
Thanks @SeanKilleen !!

**Comment by OsirisTerje** [comment]
Perhaps we should add links to the generated API docs where relevant

**Comment by SeanKilleen** [comment]
@OsirisTerje great idea on referencing the API docs -- especially once I get https://github.com/nunit/docs/issues/941 completed, which should ensure the reference version of NUnit stays up to date in the docs.

**Pull request merged**